src/pokemon.c

```diff
@@ -3367,6 +3367,20 @@ u8 CalculatePartyCount(struct Pokemon *party)
     return partyCount;
 }
 
+u8 CalculatePartyCountOfSide(u32 battler, struct Pokemon *party)
```

Comment:
@GhoulMage In case you pr in the future but we use u32 / bool32 for functions as well.

Reply:
Dang I thought this was good because the regular `CalculatePartyCount` is a u8

Both of those should be changed in a cleanup then I suppose?

Reply:
> @GhoulMage In case you pr in the future but we use u32 / bool32 for functions as well.

sure thing ^^ but as pawkkie said i did make the func based on that other one and that was `u8`, and lots of similar util funcs are u8, u16 or similar

Reply:
If you want. We usually don't do intentional clean up unless we work on the specific function. Not sure if it is worth the pr now though.

Reply:
if i push new changes to this same branch will this PR we are on reopen?

i don't mind doing another PR but i'd say this is such a small thing that's really not gonna affect anything, and party count will never exceed 6 let alone `0xFF`, idk if this is about the comparison with chosenMonId but that variable is s32 and not u32. tell me if you want it done though and i will asap

Reply:
It doesn't need to be changed. I commented just in case for the future but I also don't think it is worth changing now.

Reply:
oh yeah, will do that from now on!